shared/metadata.go

```diff
@@ -54,12 +54,29 @@ func (m MetadataResults) Less(i, j int) bool { return m[i].Test < m[j].Test }
 
 // GetMetadataResponse retrieves the response to a WPT Metadata query.
 func GetMetadataResponse(testRuns []TestRun, client *http.Client, log Logger) MetadataResults {
+	var productAtRevisions = make([]ProductAtRevision, len(testRuns))
```

Comment:
I wonder about the `revision` part of this. We don't serve different metadata for different revisions, so as long as its ignored somewhere then it's not too problematic.

But, it's more effort here to try and pipe the `revision` part; why not just take the TestRun's `Product` struct?

Reply:
Because in the metadata endpoint, I use productSpecs, err = shared.ParseProductOrBrowserParams(q), thought it would be clean to provider an API using ProductAtRevision